scripts/post-process-css/formatters/filter-typography-tokens/filter-typography-tokens.ts

```diff
@@ -0,0 +1,51 @@
+import { ContextTokens } from "../../context-tokens";
+import { CssProperty } from "../../css-parser/css-parser.types";
+import { Decorator } from "../decorator";
+
+export class FilterTypographyTokens extends Decorator {
```

Comment:
I'm also adding renaming of the tokens so that there is an actual smooth transition possible from frozenproduct to product.

Reply:
This is done now. However I also added a sanity check given the assumption that's been made that each token has a mirror adaptive/responsive token. This doesn't appear to be 100% true currently, but could be a config issue.